src/target_intacct/statistical_journal.py

```diff
@@ -92,6 +92,48 @@ def load_statistical_journal_entries(
     return journal_entries
 
 
+def build_entry(
+    row,
+    employee_ids,
+    class_ids,
+    location_ids,
+    department_ids,
+    object_name,
+    accountNo,
+):
+    if accountNo == 98051:
```

Comment:
Is this account number something specific to Fresh or something that would be useful to anyone using this Target?

Reply:
I'm not sure, let me check with Zoe

Reply:
Thanks. If this is a Fresh-specific thing can we implement this conditional in the tap (e.g. in the query)? Or this might be a good place to introduce Transformations and make this a transformation? https://docs.meltano.com/guide/transformation